src/Sql/dbo/Stored Procedures/Organization_ReadManyByIds.sql

```diff
@@ -0,0 +1,68 @@
+CREATE PROCEDURE [dbo].[Organization_ReadManyByIds]
```

Comment:
Can we add a simple happy path integration test for this stored proc to ensure that EF query and the stored proc work properly?